docs/docs/kognic-io/coordinate_systems/index.html

```diff
@@ -133,8 +133,8 @@
/p><table><thead><tr><th align="left">Type of data</th><th align="left">Single-lidar</th><th align="left">Multi-lidar</th></tr></thead><tbody><tr><td align="left">Ego poses &amp; IMU data (rotation)</td><td align="left">Lidar</td><td align="left">Reference</td></tr<tr><td align="left">Ego poses &amp; IMU data (position)</td><td align="left">World</td><td align="left">World</td></tr><tr><td align="left">OpenLABEL export <br>3D geometries</td><td align="left">Lidar</td><td align="left">Reference</td></tr><tr><td align="left">OpenLABEL export <br>2D geometries</td><td align="left">Pixel</td><td align="left">Pixel</td></tr><tr><td align="left">Pre-annotations <br>3D geometries</td><td align="left">Lidar</td><td align="left">Reference</td></tr><tr><td align="left">Pre-annotations <br>2D geometries</td><td align="left">Pixel</td><td align="left">Pixel</td></tr></tbody></table></div><footer class="theme-doc-footer docusaurus-mt-lg"><div class="theme-doc-footer-edit-meta-row row"><div class="col"><a href="https://github.com/annotell/public-docs/docs-src/docs/kognic-io/coordinate_systems.md" target="_blank" rel="noreferrer noopener" class="theme-edit-this-page"><svg fill="currentColor" height="20" width="20" viewBox="0 0 40 40" class="iconEdit_Z9Sw" aria-hidden="true"><g><path d="m34.5 11.7l-3 3.1-6.3-6.3 3.1-3q0.5-0.5 1.2-0.5t1.1 0.5l3.9 3.9q0.5 0.4 0.5 1.1t-0.5 1.2z m-29.5 17.1l18.4-18.5 6.3 6.3-18.4 18.4h-6.3v-6.2z"></path></g></svg>Edit this page</a></div><div class="col lastUpdated_vwxv"></div></div></footer></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><a class="pagination-nav__link pagination-nav__link--prev" href="/docs/kognic-io/calibrations/cameras-custom"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">Custom Camera Calibrations</div></a><a class="pagination-nav__link pagination-nav__link--next" href="/docs/kognic-io/error_handling"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Errors</div></a></nav></div></div><div class="col col--3"><div class="tableOfContents_bqdL thin-scrollbar theme-doc-toc-desktop"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#the-reference-coordinate-system-and-calibrations" class="table-of-contents__link toc-highlight">The reference coordinate system and calibrations</a></li><li><a href="#the-world-coordinate-system-and-ego-motion-data" class="table-of-contents__link toc-highlight">The world coordinate system and ego motion data</a></li><li><a href="#single-lidar-case" class="table-of-contents__link toc-highlight">Single-lidar case</a></li><li><a href="#multi-lidar-case" class="table-of-contents__link toc-highlight">Multi-lidar case</a></li><li><a href="#different-coordinate-systems-for-different-kinds-of-data" class="table-of-contents__link toc-highlight">Different coordinate systems for different kinds of data</a></li></ul></div></div></div></div></main></div></div></div>
```

Comment:
There's some error with the line breaks here.  The "Ego poses & IMU data (position)	World	World" text is on the same line as the one before.